javascript/ql/lib/semmle/javascript/Expr.qll

```diff
@@ -379,7 +379,10 @@ class NullLiteral extends @null_literal, Literal { }
  * false
  * ```
  */
-class BooleanLiteral extends @boolean_literal, Literal { }
+class BooleanLiteral extends @boolean_literal, Literal {
+  /** Gets the value of this literal. */
+  boolean getBoolValue() { if this.getRawValue() = "true" then result = true else result = false }
```

Comment:
Maybe consider naming consistency with TypeInference.qll:

```ql
  /** Gets the unique Boolean value that this node evaluates to, if any. */
  boolean getTheBooleanValue() { ... }
  ```

Reply:
`getBoolValue` is consistent with the predicate in `YamlBool` (In `YAML.qll`).   
So I prefer to keep it as is.